statchatexpress/app.js

```diff
@@ -3,12 +3,30 @@ import cors from "cors";
 import morgan from "morgan";
 import { sequelize } from "./database.js";
 import { User, Stats } from "./models/index.js";
+import session from "express-session";
 
 const app = express();
-
-app.use(cors());
+const userSecret = process.env.SESSION_SECRET;
+const maximumAge = parseInt(process.env.SESSION_MAX_AGE);
```

Comment:
Thanks for remembering to hide these tokens! It's actually fine to hardcode max age, since we normally want  that to be the same for everyone.